plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -79,6 +79,16 @@ const (
 // For example:
 // https://github.com/kubernetes/kubernetes/blob/36a218e/plugin/pkg/scheduler/factory/factory.go#L422
 
+var (
+	predicatesOrdering = []string{"CheckNodeCondition",
```

Comment:
It would be great to use constants instead of string literals here.

Reply:
Yes that is the plan.